rust/macros/lib.rs

```diff
@@ -166,3 +170,79 @@ pub fn vtable(attr: TokenStream, ts: TokenStream) -> TokenStream {
 pub fn concat_idents(ts: TokenStream) -> TokenStream {
     concat_idents::concat_idents(ts)
 }
+
+/// Used to specify the pinning information of the fields of a struct.
+///
+/// This is somewhat similar in purpose as
+/// [pin-project-lite](https://crates.io/crates/pin-project-lite).
+/// Place this macro on a struct definition and then `#[pin]` in front of the attributes of each
+/// field you want to have structurally pinned.
```

Comment:
To me this reads strange.

Reply:
Maybe “field you want to structurally pin.”

Reply:
Ah, I was parsing "(... in front of the attributes of each field you want) to have them structurally pinned" (i.e. them = the fields), but I see what you mean now, it makes sense.

Gary's suggestion is good, otherwise just ignore this one.